external-api/src/cron/eStoreDealFolderCreationJob.cron.ts

```diff
@@ -42,18 +45,10 @@ export const eStoreDealFolderCreationJob = async (eStoreData: Estore) => {
 
       // check if there are any facilityIds
       if (eStoreData.facilityIdentifiers.length) {
-        // add a new job to the `Cron Job Manager` to create the Facility Folders for the current Deal
-        const facilityCreationTimer = addMinutes(new Date(), 12);
-        eStoreCronJobManager.add(`Facility${eStoreData.dealId}`, facilityCreationTimer, () => {
-          eStoreFacilityFolderCreationJob(eStoreData);
-        });
-        console.info('Cron task started: Create the Facility folder');
-        eStoreCronJobManager.start(`Facility${eStoreData.dealId}`);
+        // Create eStore Facility directories
       }
```

Comment:
Assume this is going to be implemented in another PR

```suggestion
```

Reply:
Correct with additional assertions.